Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -0,0 +1,65 @@
+import org.opensim.modeling.*;
+import java.io.IOException;
+import static java.nio.file.StandardOpenOption.*;
+import java.nio.file.*;
+
+class TestEditMarkers {
+  public static void main(String[] args) {
+    try {
+        // GUI model loading
+        Model model = new Model("gait10dof18musc_subject01.osim");
+        OpenSimContext context=null;
+        context = new OpenSimContext(model.initSystem(), model);
+        MarkerSet markerset = model.getMarkerSet();
+        // The following code exercises the Marker->Add New  functionality
+        Vec3 offset = new Vec3(0.11, 0.22, 0.33);
+        Body body = model.getBodySet().get(0);
+        String newMarkerName = "newMarker";
+        Marker newMarker = new Marker();
+        newMarker.setName(newMarkerName);
+        newMarker.set_location(offset);
+        newMarker.setParentFrame(body);
+        context.cacheModelAndState();
+        markerset.adoptAndAppend(newMarker);
+        try {
+            context.restoreStateFromCachedModel();
+        } catch (IOException ex) {
+            System.exit(1);
+        }
+        // This exercises Marker -> delete
+        context.cacheModelAndState();
+        // we use getComponent to make sure Marker is accessible thru model traversal
```

Comment:
I definitely like the idea of having traversal tests from the GUI. Please do add that as test case.

I don't believe it needs to be exercised here simply to be exercised. Use the MarkerSet. It would be a different story if we didn't already access the MarkerSet.